src/main/webapp/person.jsp

```diff
@@ -20,445 +19,463 @@
-</jsp:include>
-	</tbody>
-</table>
+                        <td width="450"><jsp:include page="inc.erzeugeFormular.jsp">
+                                <jsp:param name="ID" value="<%= id%>" />
+                                <jsp:param name="Formular" value="person" />
+                                <jsp:param name="Datenfeld" value="Identifizierungsproblem" />
+                                <jsp:param name="cols" value="40" />
+                                <jsp:param name="rows" value="5" />
+                            </jsp:include></td>
+                    </tr>
+                    <jsp:include page="inc.modul.jsp">
+                        <jsp:param name="ID" value="<%= id%>" />
+                        <jsp:param name="Formular" value="person" />
+                        <jsp:param name="Modul" value="namen" />
+                    </jsp:include>
+                    <tr>
+                        <td width="200">
+                            <% Language.printDatafield(out, session, formular, "GND");%>
+                        </td>
+                        <td width="450"><jsp:include page="inc.erzeugeFormular.jsp">
+                                <jsp:param name="ID" value="<%= id%>" />
+                                <jsp:param name="Formular" value="person" />
+                                <jsp:param name="Datenfeld" value="GND" />
+                                <jsp:param name="size" value="50" />
+                            </jsp:include>
 
-<div id="tab1">
-<div id="header">
-<ul id="primary">
-	<li><span>
-            <% Language.printTextfield(out,session, formular,"TabZusatz");%>
-            </span></li>
-	<li><a href="javascript:onoff('tab2','tab1');">
-            <% Language.printTextfield(out,session, formular,"TabASW");%>
-            </a></li>
-	<li><a href="javascript:onoff('tab3','tab1');">
-            <% Language.printTextfield(out,session, formular,"TabEthnieAreal");%>
-            </a></li>
-	<li><a href="javascript:onoff('tab4','tab1');">
-            <% Language.printTextfield(out,session, formular,"TabVerwandte");%>
-            </a></li>
-	<li><a href="javascript:onoff('tab5','tab1');">
-            <% Language.printTextfield(out,session, formular,"TabEinzelbelege");%>
-            </a></li>
-	<li><a href="javascript:onoff('tab6','tab1');">
-            <% Language.printTextfield(out,session, formular,"TabBemerkungen");%>
-            </a></li>
-</ul>
-</div>
-<div id="main"><jsp:include page="inc.erzeugeFormular.jsp">
-	<jsp:param name="ID" value="<%= id %>" />
-	<jsp:param name="Formular" value="person" />
-	<jsp:param name="Datenfeld" value="QuiEt" />
-</jsp:include>
-</div>
-</div>
+                            <jsp:include page="inc.erzeugeFormular.jsp">
+                                <jsp:param name="ID" value="<%=id%>" />
+                                <jsp:param name="Formular" value="person" />
+                                <jsp:param name="Datenfeld" value="GNDLink" />
```

Comment:
Do we really need to make a different call / define a different GNDLink here? I think it might be better to just generate the link when the GND field is shown.

Reply:
Ah ok now i understand ... the GND is basically a regular textfield and the GNDlink is only rendered in addition... ok then we can leave it this way, but i guess we should rename the template files to make things clearer